code/game/objects/items/weapons/power_cells.dm

```diff
@@ -95,7 +95,7 @@
 	name = "infinite-capacity power cell!"
 	icon_state = "icell"
 	origin_tech =  null
-	maxcharge = 30000
+	maxcharge = 1000000000 //Good luck charging this, buff it if it gets fully charged
```

Comment:
It is, as ```1.#INF```, but the ```INFINITY``` value is just a ```#define``` that was made ages ago, it's not a built-in.